android/anc/src/test/java/org/smartregister/fhircore/anc/ui/anccare/details/AncDetailsFragmentTest.kt

```diff
@@ -156,4 +166,140 @@ internal class AncDetailsFragmentTest : FragmentRobolectricTest() {
6006\",\"display\":\"Pregnant\"}]},\"text\":{\"status\":\"generated\",\"div\":\"<div xmlns=\\\"http://www.w3.org/1999/xhtml\\\"><div class=\\\"hapiHeaderText\\\">Patient two <b>MOM TWO </b></div><table class=\\\"hapiPropertyTable\\\"><tbody><tr><td>Address</td><td><span>Nairobi </span></td></tr><tr><td>Date of birth</td><td><span>17 September 1985</span></td></tr></tbody></table></div>\"},\"extension\":[{\"url\":\"http://hl7.org/fhir/StructureDefinition/flag-detail\",\"valueString\":\"Pregnant\"}],\"identifier\":[{\"use\":\"official\",\"value\":\"100013\"}],\"active\":true,\"name\":[{\"family\":\"Mom two\",\"given\":[\"Patient two\"]}],\"telecom\":[{\"system\":\"phone\",\"value\":\"21212121212\"}],\"gender\":\"female\",\"birthDate\":\"1985-09-17\",\"address\":[{\"city\":\"Nairobi\",\"district\":\"Rahim Yar Khan\"}],\"link\":[{\"other\":{\"reference\":\"Patient/e8725b4c-6db0-4158-a24d-50a5ddf1c2ed\"},\"type\":\"refer\"}]},\"search\":{\"mode\":\"match\"}},{\"fullUrl\":\"http://fhir.labs.smartregister.org/fhir/Goal/d1853316-17fd-4ebb-96cb-8120a38bdf9e\",\"resource\":{\"resourceType\":\"Goal\",\"id\":\"d1853316-17fd-4ebb-96cb-8120a38bdf9e\",\"meta\":{\"versionId\":\"1\",\"lastUpdated\":\"2021-09-17T12:51:04.693+00:00\",\"source\":\"#ba5f8082a49ee86b\"},\"lifecycleStatus\":\"active\",\"subject\":{\"reference\":\"Patient/e8725b4c-6db0-4158-a24d-50a5ddf1c2ed\"}},\"search\":{\"mode\":\"match\"}},{\"fullUrl\":\"http://fhir.labs.smartregister.org/fhir/Condition/23390fe8-a48d-42a6-98bd-eadbca0aa33f\",\"resource\":{\"resourceType\":\"Condition\",\"id\":\"23390fe8-a48d-42a6-98bd-eadbca0aa33f\",\"meta\":{\"versionId\":\"1\",\"lastUpdated\":\"2021-09-17T12:51:03.642+00:00\",\"source\":\"#e7d882277be61c18\"},\"clinicalStatus\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/condition-clinical\",\"code\":\"active\"}]},\"verificationStatus\":{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/condition-ver-status\",\"code\":\"confirmed\"}]},\"category\":[{\"coding\":[{\"system\":\"http://terminology.hl7.org/CodeSystem/condition-category\",\"code\":\"problem-list-item\",\"display\":\"Problem List Item\"}]}],\"code\":{\"coding\":[{\"system\":\"http://example.org/tbd\",\"code\":\"TBD\",\"display\":\"Pregnancy\"}],\"text\":\"Pregnancy\"},\"subject\":{\"reference\":\"Patient/e8725b4c-6db0-4158-a24d-50a5ddf1c2ed\"},\"onsetDateTime\":\"2021-07-06\"},\"search\":{\"mode\":\"match\"}},{\"fullUrl\":\"http://fhir.labs.smartregister.org/fhir/Practitioner/1747\",\"resource\":{\"resourceType\":\"Practitioner\",\"id\":\"1747\",\"meta\":{\"versionId\":\"1\",\"lastUpdated\":\"2021-09-06T05:44:24.513+00:00\",\"source\":\"#36856d49828faa6a\"}},\"search\":{\"mode\":\"match\"}},{\"fullUrl\":\"http://fhir.labs.smartregister.org/fhir/Encounter/2015\",\"resource\":{\"resourceType\":\"Encounter\",\"id\":\"2015\",\"meta\":{\"versionId\":\"1\",\"lastUpdated\":\"2021-09-17T13:03:38.816+00:00\",\"source\":\"#d844fa2fe47c0963\"}},\"search\":{\"mode\":\"match\"}}]}"
```

Comment:
can we put this in a fixture flat file then load it in?

Reply:
@pld This is done